docs/backend/modules/importer.md

```diff
@@ -5,46 +5,26 @@
 This module is responsible for managing imports to ergonode from external systems or files. 
 
 
-## Reader
+## Custom Import error
 
-**Reader** is a component responsible for reading files in different formats. 
+If an error occurs during the import process, we may want to inform the user by presenting the relevant content in the import error list.
 
-One Reader class is responsible for one file type.
-
-To use it you need to create a class which implements **FileReaderInterface**.
+For that we need to create custom error. We're doing that by creating new exception class extended from ```Ergonode\Importer\Infrastructure\Exception\ImportException```
```

Comment:
```YourNameSpace\Infrastructure\Exception\ImportException```

Reply:
its ok this is importer module exception